gallery/gallery/lib/pages/demo.dart

```diff
@@ -179,15 +179,19 @@ class _DemoPageState extends State<DemoPage> with TickerProviderStateMixin {
     final colorScheme = Theme.of(context).colorScheme;
     final iconColor = colorScheme.onSurface;
     final selectedIconColor = colorScheme.primary;
+    final appBarPadding = 20.0;
```

Comment:
Only do this if it is desktop, otherwise it'll look strange on mobile

Reply:
Good call